pkg/base1/test-dbus-common.js

```diff
@@ -239,30 +239,7 @@ export function common_dbus_tests(channel_options, bus_name) { // eslint-disable
                 });
     });
 
-    QUnit.test("bad variants", function (assert) {
```

Comment:
Can you move the deduplication to a separate commit?

Reply:
Done.